src/lib/multicall/Multicall.ts

```diff
@@ -372,10 +332,6 @@ export class Multicall {
       isAlchemy: isFallbackMode,
```

Comment:
we still have isAlchemy + isFallback which mean same thing? let's pick only one?

Reply:
Renamed everything to `isAlchemy`